asreviewcontrib/semantic_clustering/semantic_clustering.py

```diff
@@ -86,8 +80,8 @@ def SemanticClustering(asreview_data_object, output_file):
     labels = run_KMeans(tsne, n_clusters, 10)
 
     # visualize clusters
-    print("Visualizing clusters...")
-    _visualize_clusters(tsne, labels)
+    # print("Visualizing clusters...")
+    # _visualize_clusters(tsne, labels)
```

Comment:
Are you sure you want to comment this?

Reply:
Yes! The interactive dashboard works well enough to not need this anymore. Might remove this part of the code altogether.

Reply:
Oké, lets remove then.